typerighter.cfn.yaml

```diff
@@ -100,6 +100,8 @@ Resources:
           include "application"
           play.http.secret.key="${PlaySecret}"
           typerighter.ngramPath="/opt/ngram-data"
+          typerighter.sheetId="1bLCD7CrGMDAuuk76e0MTF1-DUNgmXrvTc1d5n2UMZR8"
```

Comment:
As above, probably best not to be in an open repo

Reply:
also as above 👍